diff --git a/sdk/identity/azure-identity/vcpkg/Config.cmake.in b/sdk/identity/azure-identity/vcpkg/Config.cmake.in
index 92a29bb9..40bfcbfb 100644
--- a/sdk/identity/azure-identity/vcpkg/Config.cmake.in
+++ b/sdk/identity/azure-identity/vcpkg/Config.cmake.in
@@ -7,7 +7,6 @@ include(CMakeFindDependencyMacro)
 find_dependency(azure-core-cpp)
 
 if(WIN32 AND NOT (CMAKE_SYSTEM_NAME STREQUAL "WindowsStore" AND CMAKE_SYSTEM_VERSION STREQUAL "10.0"))
-  find_dependency(wil)
 else()
   find_dependency(OpenSSL)
 endif()
